sys/net/routing/rpl/rpl.c

```diff
@@ -116,6 +110,11 @@ uint8_t rpl_init(int if_id, ipv6_addr_t *address)
     return SIXLOWERROR_SUCCESS;
 }
 
+kernel_pid_t rpl_get_if(void)
```

Comment:
Ah okay… I was confused because the old stack uses just plain integers as interfaces (which are enumerated 0…N; because there are no extra threads for interfaces, just the transceiver thread) while the new stack just uses the PID of the MAC layer.